airship_tempest_plugin/tests/api/shipyard/test_actions.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 AT&T Corp
```

Comment:
Did you run pep8?

Reply:
Yes, pep8 was run and didn't report any failures around these lines